.pre-commit-config.yaml

```diff
@@ -1,12 +1,21 @@
 exclude: '^(.*egg.info.*|.*/parameters.py|docs/).*$'
+# TODO: SNOW-1332230: Remove files and fix associated linting issues
```

Comment:
Is it possible to remove this file range now?

Reply:
why it will trigger changes in other files?

Reply:
Ok, I think I know the reason. The changes are not due to this file range but due to the change on line 18, which changed the expected python syntax from 38 to 39. 

I think we should keep it as py38 for now.

Reply:
Oh did not realize that, okay will change back to py38.